actions/views/root.js

```diff
@@ -47,21 +47,29 @@ export function unregisterPluginTranslationsSource(pluginId) {
     Reflect.deleteProperty(pluginTranslationSources, pluginId);
 }
 
-export function loadTranslations(locale, url) {
+export async function loadTranslations(locale, url) {
     return (dispatch) => {
         const translations = {};
+        var loc;
         Object.values(pluginTranslationSources).forEach((pluginFunc) => {
             Object.assign(translations, pluginFunc(locale));
         });
 
-        // No need to go to the server for EN
         if (locale === 'en') {
-            copyAndDispatchTranslations(dispatch, translations, en, locale);
-            return;
+            loc = en;
+        } else {
```

Comment:
since we're initializing to `en` maybe we don't need an else case here, only an `if`


```
if (locale !== 'en') {
	try {
	  	const serverTranslations = await Client4.getTranslations(url);
		Object.assign(translations, serverTranslations);
	} catch (error) {
	  	console.error(error); //eslint-disable-line no-console
	}
}